spec/features/audit/allocation_spec.rb

```diff
@@ -104,4 +104,19 @@
     expect(page).to have_select("allocate_to", selected: "No one")
     expect(page).to have_content("1 items unallocated")
   end
+
+  scenario 'Allocate all content within current page', :js do
+    create_list(:content_item, 100)
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/290/commits/d4fd166a4ef59282c85bf2d178a0748e0c77dbf3#diff-69c737006735f69541cec3ef0afaea49R109